src/main/java/com/tstepnik/planner/domain/user/UserDto.java

```diff
@@ -0,0 +1,47 @@
+package com.tstepnik.planner.domain.user;
+
+import javax.validation.constraints.NotNull;
+
+public class UserDto {
+
+    @NotNull
```

Comment:
Is this actually making any sense? Do we only require user to always have just login?

Reply:
I added annotation to most important fields: login, password and email. Other can be null.